docs/pages/10_view.adoc

```diff
@@ -0,0 +1,629 @@
+
+=== View field
+
+A view field is a graphical component within a view that is designed to represent and interact with a single, fundamental piece of information. It is typically associated with primitive data types such as text or numbers and is used to either display or capture information from users. The full list of modifiers can be found in the xref:appx_syntax.adoc#view-field[Appendix].
+
+The syntax for defining a view field is as follows:
+
+*Syntax:*
+
+[subs="quotes"]
+----
+[purple]##**field**## &lt;primitive> &lt;name> &#x5B;[purple]##**<=**## &lt;expression>]
+    &#x5B;[purple]##**required:**##([purple]##**true**##|[purple]##**false**##)]
+    &#x5B;[purple]##**enabled:**##&lt;enabled>]
+    &#x5B;[purple]##**default:**##&lt;default>]
+    &#x5B;[purple]##**set:**##([purple]##**true**##|[purple]##**false**##)]
+
+    &#x5B;[purple]##**label:**##&lt;label>]
+    &#x5B;[purple]##**icon:**##&lt;icon>]
+    &#x5B;[purple]##**hidden:**##&lt;hidden>]
+    &#x5B;[purple]##**width:**##&lt;width>]
+    &#x5B;[purple]##**lines:**##&lt;lines>][purple]#**;**#
+----
+
+View fields focus on how users perceive and interact with information, prioritizing user experience and visual presentation. They are distinct from transfer fields, which primarily focus on the efficient movement and sharing of data. To get specific details about this shared syntax, please refer to the section on the xref:07_transfer.adoc#field[Transfer field].
+
+Modifiers such as `label`, `icon`, `hidden`, and `width` can be used to customize the common appearance and behavior of the component. See more about them in <<Common visual modifiers>> chapter.
+
+The `lines` modifier specifies the height of the field in terms of the number of text lines it occupies. This modifier can only be used if the <primitive> data type associated with the field is a string. It controls the vertical space allocated to the field, allowing for multiline input or display of text.
+
+Here's an example of a view field definition:
+
+*Example:*
+
+[source,jsl]
+----
+field Integer price <= customer.cart.price
+    label:"Price ($)"
+    icon:"currency-usd"
+    width:4;
+----
+
+In this example, the price field is associated with the integer data type and represents the price of items in a customer's shopping cart. The label modifier specifies the text "Price ($)" to be displayed next to the field, and the icon modifier sets an icon representing currency next to the field. The width modifier determines the logical width of the field on the screen.
+
+=== View link
+
+The view link is a visual element in a user interface that represents a connection to another view, indicating a relationship between different pieces of information. It enables users to navigate between related views and interact with associated data. The full list of modifiers can be found in the xref:appx_syntax.adoc#view-link[Appendix].
```

Comment:
I can't find the specification for selector dialogs on links. Or the visual expectation of value selection / change.

Reply:
fixed